src/lib/__tests__/utils-test.js

```diff
@@ -937,34 +937,47 @@ describe('utils.deprecate', () => {
 });
 
 describe('utils.warn', () => {
-  it('expect to print a warn message', () => {
+  it('prints a warn message with a false condition', () => {
     const message = 'message';
     const warn = jest.spyOn(global.console, 'warn');
```

Comment:
might be shorter if you factorize the mocking and the resetting in a beforeEach & afterEach